cosmoz-moneyhelper-behavior.html

```diff
@@ -112,6 +112,30 @@
 	 * @demo
 	 */
 	Cosmoz.MoneyHelperBehavior = {
+		/**
+		 * Compare amounts.
+		 * @param {object} amount1 Amount 1.
+		 * @param {object} amount2 Amount 2.
+		 * @param {number} precision Decimal precision, defaults to 2 decimals.
+		 * @returns {boolean} Comparison result.
+		 */
+		amountEquals(amount1, amount2, precision = 2) {
+			return this.round(amount1.amount, precision) === this.round(amount2.amount, precision) &&
```

Comment:
Needs null-checks for amount1 and amount2 - use `this.isAmount()`!

Reply:
Updated with use of that function.